controller/workitem.go

```diff
@@ -72,8 +72,14 @@ func (c *WorkitemController) List(ctx *app.ListWorkitemContext) error {
 		return jsonapi.JSONErrorResponse(ctx, errors.NewBadParameterError("could not parse filter", err))
 	}
 	if ctx.FilterAssignee != nil {
-		exp = criteria.And(exp, criteria.Equals(criteria.Field("system.assignees"), criteria.Literal([]string{*ctx.FilterAssignee})))
-		additionalQuery = append(additionalQuery, "filter[assignee]="+*ctx.FilterAssignee)
+		if *ctx.FilterAssignee == "none" {
```

Comment:
Also, @hectorj2f do we have any restrictions on usernames? Can somebody for example call oneself "none" as a username? That would be very odd.